msteams-platform/m365-apps/prerequisites.md

```diff
@@ -44,44 +43,24 @@ You also need to enable sideloading for your tenant:
-
-> [!IMPORTANT]
-> It can take up to five days after creating a [Microsoft 365 developer sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) and enrolling in [Microsoft 365 Targeted Releases](#enroll-your-developer-tenant-for-microsoft-365-targeted-releases) for sideloaded Teams apps to appear in Outlook and Microsoft 365 app.
-
-This step is required to preview your Teams app running in Outlook (desktop and web); the Microsoft 365 app (desktop and web) supports Teams tab apps in both Standard and Targeted release channels. To enroll your test tenant for Microsoft 365 Targeted Releases:
-
-1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com) with your test tenant credentials.
-1. Go to **Settings** > **Org Settings** > **Organization profile**.
-1. Select **Release preferences**.
-1. Select any *Targeted Release* preference:
-    1. **Targeted Release for everyone**
-    1. **Targeted Release for select users**
-
-    :::image type="content" source="images/m365-admin-center-targeted-releases.png" alt-text="The screenshot is an example that shows the Microsoft 365 admin center 'Release preferences' menu with Targeted Release option selected.":::
-
-1. Select **Save**.
-
-For more information on Microsoft 365 release options, see [set up the Standard or Targeted Release options](/microsoft-365/admin/manage/release-options-in-office-365?view=o365-worldwide&preserve-view=true#targeted-release) in *Microsoft 365 admin center help*.
-
 ## Install Microsoft 365 apps in your test environment
 
 ### Desktop
 
-You can preview Teams apps running in Outlook on Windows desktop by using a recent *Beta Channel build*. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 Beta Channel build.
+You can preview Teams apps running in Outlook on Windows desktop by using a recent *Channel build*. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 [value] Channel build.
 
-To install Microsoft 365 Beta Channel applications in your test environment:
+To install Microsoft 365 applications in your test environment:
 
 1. Sign in to your test environment with your test tenant credentials.
 1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117).
 1. Select a local folder to store the extracted files.
-1. Go to the local folder and open *configuration-Office365-x86.xml* (or **x64.xml*, depending on your environment) in a text editor and update the *Channel* value to `BetaChannel`.
+1. Go to the local folder and open *configuration-Office365-x86.xml* (or **x64.xml*, depending on your environment) in a text editor and update the *Channel* value to `[what value]`.
```

Comment:
*CurrentChannel* is the default in the XMLs that come with the tool. So just say "and ensure the *Channel* value is `Current`.